region/wreckedship/main/Sponge Bath.json

```diff
@@ -242,8 +242,8 @@
       "name": "10 Power Bomb Crystal Flash",
       "requires": [
         "f_DefeatedPhantoon",
+        {"resourceAvailable": [{"type": "RegularEnergy", "count": 50}]},
```

Comment:
If you have 50 energy, you are not in health bomb and can kill the bull with the PB performing the 10PB CF.
resetRoom is for farming energy to ensure you are not in healthbomb.